src/test/java/org/takes/rq/RqHeadersTest.java

```diff
@@ -80,4 +80,50 @@ public void findsAllHeaders() throws IOException {
         );
     }
 
+    /**
+     * RqHeaders can find single header.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void findsSingleHeader() throws IOException {
+        MatcherAssert.assertThat(
+            new RqHeaders.Smart(
+                new RqHeaders.Base(
+                    new RqFake(
+                        Arrays.asList(
+                            "GET /g",
+                            "Host: www.takes.com"
+                        ),
+                        ""
+                    )
+                )
+            // @checkstyle MultipleStringLiteralsCheck (1 line)
```

Comment:
@erimerturk instead of adding this suppression, change the header key from `host` to something else that is unique to this test.